hail/python/hail/matrixtable.py

```diff
@@ -603,6 +605,14 @@ def __init__(self, mir):
                                     self._entry.items()):
             self._set_field(k, v)
 
+    @property
+    def _schema(self) -> tmatrix:
```

Comment:
This should be flipped eventually (row type coming from matrix/table type) but that can come later.

Reply:
Yes, I'm planning on that.